src/frontend/src/flows/register/passkey.ts

```diff
@@ -0,0 +1,94 @@
+import { withLoader } from "$src/components/loader";
+import { mainWindow } from "$src/components/mainWindow";
+import { toast } from "$src/components/toast";
+import { I18n } from "$src/i18n";
+import { IIWebAuthnIdentity } from "$src/utils/iiConnection";
+import { renderPage, TemplateElement } from "$src/utils/lit-html";
+import { unknownToString } from "$src/utils/utils";
+import { constructIdentity } from "$src/utils/webAuthn";
+import { isCancel, webAuthnErrorCopy } from "$src/utils/webAuthnErrorUtils";
+import { html, TemplateResult } from "lit-html";
+
+import copyJson from "./passkey.json";
+
+/* Anchor construction component (for creating WebAuthn credentials) */
+
+const savePasskeyTemplate = ({
+  construct,
+  i18n,
+}: {
+  construct: () => void;
+  i18n: I18n;
+}): TemplateResult => {
+  const copy = i18n.i18n(copyJson);
+  const slot = html`
+    <hgroup>
+      <h1 class="t-title t-title--main">${copy.save_passkey}</h1>
+      <p class="t-paragraph">
+        ${copy.select} <strong class="t-strong">${copy.save_passkey}</strong> ${
+    copy.and_complete_prompts
+  }</p>
+    </hgroup>
+    <button @click=${() =>
+      construct()} data-action="construct-identity" class="c-button">${
+    copy.save_passkey
+  }</button>
+    <p class="t-paragraph">
```

Comment:
a paragraph should only contain text. I would wrap the whole "copy" part of this screen in a `section.c-marketing-block` and make `aside.l-stack` with  the different section not paragraphs.

Reply:
instead of `<strong class="t-strong">` you can just use a h3 or something with the `t-title` class

Reply:
Thanks a lot!